framework_tvb/tvb/interfaces/web/templates/jinja2/form_fields/datatype_select_field.html

```diff
@@ -1,14 +1,28 @@
-<p class="field-data">
+<div class="field-data">
```

Comment:
why this change from `p` into `div` is necessary ?
it will break consistency with the other fields...

Reply:
Well, when I did that the div added by me didn't want to appear where I wanted and I read on the internet that block elements such as div  should be inside of inline elements such as paragraph (not that it can't be done, it's just not recommented) and changing it to div helped my problem, but now I tried again with <p> and for some reason now it works, so I guess we can leave it as a paragraph.